kms/api-client/asymmetric.py

```diff
@@ -86,46 +115,51 @@ def encryptRSA(plaintext, client, key_path):
 
 
 # [START kms_sign_asymmetric]
-def signAsymmetric(message, client, key_path):
+def sign_asymmetric(message, key_name):
```

Comment:
The only key creator is for RSA encryption keys, not for RSA signing nor EC signing. So how will a user exercise this (and following) snippets?